packages/ocaml-variants/ocaml-variants.4.02.3+BS/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "4.02.3+BuckleScript patches"
+maintainer: "platform@lists.ocaml.org"
+depends: [
+  "ocaml" {= "4.02.3" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+  "base-ocamlbuild" {post}
+]
+conflict-class: "ocaml-core-compiler"
+flags: compiler
+setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build: [
+  ["./configure" "-prefix" prefix "-with-debug-runtime"]
+    {os != "openbsd" & os != "freebsd" & os != "macos"}
+  [
+    "./configure"
+    "-prefix"
+    prefix
+    "-with-debug-runtime"
+    "-cc"
+    "cc"
+    "-aspp"
+    "cc -c"
+  ] {os = "openbsd" | os = "freebsd" | os = "macos"}
+  [make "world.opt"]
+]
+install: [make "install"]
+url {
+  src: "https://github.com/bucklescript/ocaml/archive/4.02.3+BS.zip"
```

Comment:
is this release going to change? If it doesn't you need to add a checksum to the archive. Also maybe tar.gz would be a better choice of archive.